code/modules/projectiles/guns/projectile/automatic.dm

```diff
@@ -246,4 +240,17 @@
 	fire_sound = 'sound/weapons/Gunshot_smg.ogg'
 	can_suppress = 0
 	burst_size = 4
+	fire_delay = 1
+
+/obj/item/weapon/gun/projectile/automatic/ar
+	name = "arg"

```

Comment:
arg is a pretty shitty name, not really obvious that it's a gun. try making it capital letters or something (if the first letter is a capital letter, add /improper in front of the name)